src/main/kotlin/no/nav/hjelpemidler/oppgave/service/OpprettOppgaveForOverføring.kt

```diff
@@ -136,23 +138,25 @@ class OpprettOppgaveForOverføring(
 
             else -> {
                 log.info { "lagOpprettJournalføringsoppgaveRequest sakstype: $sakstype" }
-                if (sakstype == null) {
-                    log.info { "lagOpprettJournalføringsoppgaveRequest sakstype er null" }
-                }
+
```

Comment:
Hm, usikker her, alt annet kommer jo fra Kafka her, kunne kanskje tatt med erHast også i dette tilfellet.

Reply:
Vi ruller det ut sånn som det er nå for å få prodsatt hast, og så fikser vi til å få `erHast` via Kafka senere. Da vil hele SoknadsbehandlingDbClient og BehovsmeldingData forsvinne også.